phoenix-protocol/src/main/PTable.proto

```diff
@@ -52,11 +52,12 @@ message PColumn {
 
 message PTableStats {
   required bytes key = 1;
-  repeated bytes values = 2;
+  optional bytes guidePosts = 2;
```

Comment:
This might be a b/w compat issue. Safest would be to add guidePosts at the end and keep values as-is.